browser-bunyan/tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "target": "es6",
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictNullChecks": false,
```

Comment:
There are no linting errors in the .d.ts file. There are semicolons missing in the tests (from the library these types are for and I'd rather not alter the tests at all).

Re-running locally no linting errors are detected:

```
npm run lint -- browser-bunyan

> definitely-typed@0.0.1 lint /Users/paullockwood/Code/DefinitelyTyped
> node scripts/lint.js "browser-bunyan"

node ../node_modules/tslint/lib/tslint-cli --format stylish "**/*.d.ts"

```